librz/include/rz_bin.h

```diff
@@ -300,8 +300,8 @@ struct rz_bin_file_t {
 	struct rz_bin_xtr_plugin_t *curxtr;
 	// struct rz_bin_plugin_t *curplugin; // use o->plugin
 	RzList *xtr_data;
-	Sdb *sdb;
-	Sdb *sdb_info;
+	RZ_DEPRECATE Sdb *sdb;
```

Comment:
What is supposed to be used instead? Maybe add it as a comment?

Reply:
C